src/MCPClient/lib/settings/common.py

```diff
@@ -16,32 +16,186 @@
 # along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
 
 import ConfigParser
+import StringIO
+
+from django.core.exceptions import ImproperlyConfigured
+
+from env_configparser import EnvConfigParser
+
+
+class Config(object):
+    ENV_PREFIX = 'ARCHIVEMATICA_MCPCLIENT'
+
+    DBSETTINGS_FILE = '/etc/archivematica/archivematicaCommon/dbsettings'
+    DBSETTINGS_CONFIG = """[client]
+user = archivematica
+password = demo
+host = localhost
+database = MCP
+max_overflow = 40
+port = 3306
+engine = django_mysqlpool.backends.mysqlpool
+    """
+
+    CLIENTCONF_FILE = '/etc/archivematica/serverConfig.conf'
+    CLIENTCONF_CONFIG = """[MCPClient]
+MCPArchivematicaServer = localhost:4730
```

Comment:
The name of the options are really unfortunate. For example this one really refers to the gearman job server. Please don't put much attention to these details, this is only for backward compatibility.